packages/wpcom-checkout/src/checkout-line-items.tsx

```diff
@@ -226,14 +226,61 @@ function WPNonProductLineItem( {
 	/* eslint-enable wpcalypso/jsx-classname-namespace */
 }
 
-function GSuiteUsersList( { product }: { product: ResponseCartProduct } ) {
-	const users = product.extra?.google_apps_users ?? [];
+function EmailMeta( { product, isRenewal }: { product: ResponseCartProduct; isRenewal: boolean } ) {
+	const translate = useTranslate();
+
+	if ( isRenewal ) {
+		let numberOfMailboxes = null;
+
+		if ( isGSuiteOrExtraLicenseProductSlug( product.product_slug ) ) {
+			numberOfMailboxes = product.volume ?? null;
+		}
+
+		if ( isGoogleWorkspaceProductSlug( product.product_slug ) || isTitanMail( product ) ) {
+			numberOfMailboxes = product.current_quantity ?? null;
+		}
+
+		if ( numberOfMailboxes === null ) {
+			return;
```

Comment:
Per the general TS type failures (I'll list in a dedicated comment), I believe this needs to return `null` to match the expectations for a function component. It's probably worth specifying the return type of the function, maybe as `LineItemMeta | null`.
```suggestion
			return null;
```

Reply:
> It's probably worth specifying the return type of the function, maybe as LineItemMeta | null.

This function also returns a list of `LineItemMeta` items enclosed in a `<>` fragment, so I suspect we should rather use `JSX.Element | null` as return type here (unless @sirbrillig thinks there is a better alternative).

Reply:
`JSX.Element | null` looks good to me!